lib/crypto.js

```diff
@@ -182,7 +182,9 @@ function setFipsDisabled() {
   throw new errors.Error('ERR_CRYPTO_FIPS_UNAVAILABLE');
 }
 
-function setFipsForced() {
+function setFipsForced(val) {
+  if (val && fipsForced)
```

Comment:
Why the check on `fipsForced`? Isn't `setFipsForced` only called when `fipsForced` is truthy?

Reply:
heh, good point ;-) this only needs to check if `val` is truthy :-)